Add ID to add-track log message

garrettjstevens opened the pull request:
I'm not sure if this makes the message too long, but I thought I'd see what people think. This adds the ID to the output when running `add-track`, so that instead of
```
Added track "volvox.sort.gff3" to ./config.json
```
it would look like
```
Added track "volvox.sort.gff3" with ID "volvox.sort.gff3" to ./config.json
```

The hope is that this will reduce confusion in cases like https://github.com/GMOD/jbrowse-components/issues/2704

cmdcolin commented:
given that the default is that name===trackId the message can look a bit confusing. might be worth outputting specifically that it has name:x and trackId:y

garrettjstevens commented:
Another option could be to only specify the ID in the message if name !== trackId

garrettjstevens commented:
Here are some alternatives written out:

Current:.

    Added track "volvox.sort.gff3" to ./config.json

Alternative 1

    Added track "volvox.sort.gff3" with ID "volvox.sort.gff3" to ./config.json

Alternative 2 (different if name!==trackID)

    Added track "volvox.sort.gff3" to ./config.json
    Added track "Volvox Genes" with ID "volvox.sort.gff3" to ./config.json

Alternative 3

    Added track with name "volvox.sort.gff3" and trackId "volvox.sort.gff3" to ./config.json

Alternative 3 is probably the clearest. It's a bit longer, but I don't think it's too long.

cmdcolin commented:
I like alt 3 too :)

garrettjstevens commented:
Implemented alternative 3
